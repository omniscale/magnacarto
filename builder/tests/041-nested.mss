#country-label {
    text-face-name: "unifont";
    text-name:"''";
    text-size: 10;
    [zoom=5] {
        text-size:12;
    }
    [zoom>=4][type ='foo'] { text-name: "[ABBREV]"; }
}