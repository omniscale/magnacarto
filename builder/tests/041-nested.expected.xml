<Map srs="epsg:3857">
  <Parameters></Parameters>
  <FontSet name="fontset-1">
    <Font face-name="unifont"></Font>
  </FontSet>
  <Style name="country-label" filter-mode="first">
    <Rule>
      <!--Zoom{=5}-->
      <MaxScaleDenominator>25000000</MaxScaleDenominator>
      <MinScaleDenominator>12500000</MinScaleDenominator>
      <Filter>([type] = &#39;foo&#39;)</Filter>
      <TextSymbolizer fontset-name="fontset-1" size="12">[ABBREV]</TextSymbolizer>
    </Rule>
    <Rule>
      <!--Zoom{>=4}-->
      <MaxScaleDenominator>50000000</MaxScaleDenominator>
      <Filter>([type] = &#39;foo&#39;)</Filter>
      <TextSymbolizer fontset-name="fontset-1" size="10">[ABBREV]</TextSymbolizer>
    </Rule>
    <Rule>
      <!--Zoom{=5}-->
      <MaxScaleDenominator>25000000</MaxScaleDenominator>
      <MinScaleDenominator>12500000</MinScaleDenominator>
      <TextSymbolizer fontset-name="fontset-1" size="12">&#39;&#39;</TextSymbolizer>
    </Rule>
    <Rule>
      <TextSymbolizer fontset-name="fontset-1" size="10">&#39;&#39;</TextSymbolizer>
    </Rule>
  </Style>
  <Layer name="country-label" srs="" status="off">
    <StyleName>country-label</StyleName>
  </Layer>
</Map>